diff --git a/vendor/raylib/raygui.odin b/vendor/raylib/raygui.odin
index 559437a60..3af8ce530 100644
--- a/vendor/raylib/raygui.odin
+++ b/vendor/raylib/raygui.odin
@@ -11,7 +11,8 @@ when ODIN_OS == .Windows {
 	}
 } else when ODIN_OS == .Linux  {
 	foreign import lib {
-		"linux/libraygui.so" when RAYGUI_SHARED else "linux/libraygui.a",
+		//"linux/libraygui.so" when RAYGUI_SHARED else "linux/libraygui.a",
+		"system:raygui",
 	}
 } else when ODIN_OS == .Darwin {
 	when ODIN_ARCH == .arm64 {
diff --git a/vendor/raylib/raylib.odin b/vendor/raylib/raylib.odin
index 02bb6deea..20b19c7e5 100644
--- a/vendor/raylib/raylib.odin
+++ b/vendor/raylib/raylib.odin
@@ -117,7 +117,8 @@ when ODIN_OS == .Windows {
 		// multiple copies of raylib.so, but since these bindings are for
 		// particular version of the library, I better specify it. Ideally,
 		// though, it's best specified in terms of major (.so.4)
-		"linux/libraylib.so.550" when RAYLIB_SHARED else "linux/libraylib.a",
+		//"linux/libraylib.so.550" when RAYLIB_SHARED else "linux/libraylib.a",
+		"system:raylib",
 		"system:dl",
 		"system:pthread",
 	}
diff --git a/vendor/raylib/rlgl/rlgl.odin b/vendor/raylib/rlgl/rlgl.odin
index 6ac19695d..7182edc3a 100644
--- a/vendor/raylib/rlgl/rlgl.odin
+++ b/vendor/raylib/rlgl/rlgl.odin
@@ -135,7 +135,8 @@ when ODIN_OS == .Windows {
 		// multiple copies of raylib.so, but since these bindings are for
 		// particular version of the library, I better specify it. Ideally,
 		// though, it's best specified in terms of major (.so.4)
-		"../linux/libraylib.so.550" when RAYLIB_SHARED else "../linux/libraylib.a",
+		//"../linux/libraylib.so.550" when RAYLIB_SHARED else "../linux/libraylib.a",
+		"system:raylib",
 		"system:dl",
 		"system:pthread",
 	}
